crates/aptos-rosetta/src/lib.rs

```diff
@@ -81,6 +77,7 @@ pub fn bootstrap(
 ) -> anyhow::Result<tokio::runtime::Runtime> {
     let runtime = tokio::runtime::Builder::new_multi_thread()
         .thread_name("rosetta")
+        .disable_lifo_slot()
```

Comment:
IIUC, the lifo functionality was built into tokio scheduler as an optimization to speed up scheduling. Hopefully the performance penality from disabling is not too visible for us. Let's wait for Forge result before merging this PR.

Reply:
I don't think we're sensitive to such optimization, we're not context switch heavy applications, but sure